docs/videos/video-2-script-draft.txt

```diff
@@ -0,0 +1,19 @@
+How Event Bus Can Help You (And Veterans!)
+
+Now let's take a look at Event Bus in action, using a visual component to help see how the data flows through the system.
+
+Here we have a representation of our source data; in this case a medical claims processing event coming from a VA system. Our Veteran Mark has just had a medical procedure, and the event that the claim has been processed for procedure is about to be available. In the event data is relevant information that can help Mark if other VA systems know about it.
+
+This demo allows us to choose what data is included in the medical event; we'll select that the event requires a new prescription, and that the health update was important and Mark should get an SMS update about it. Now let's trigger that medical processing event!
```

Comment:
Should we say something here about choosing the data in the event is for demo purposes only, and in real life events will always be generated by a production system with a defined structure and payload?

Reply:
Yeah I'll adjust this to say something like "In a real situation, this comes pre-defined from the provider, but for the purposes of this demo..." or something.